contractsdk/cpp/reserved/unified_check.cc

```diff
@@ -238,6 +241,24 @@ DEFINE_METHOD(UnifiedCheck, identity_check) {
     ctx->ok("success");
 }
 
+// identity_query return whether address is in identity list
+// keep this method for convenience
+DEFINE_METHOD(UnifiedCheck, identity_query) {
```

Comment:
Is it necessary to add an extra function of ban_query?

Reply:
no need, banned_check accepts "contract" argument.